From c8ed11522b4acbe378687b6388ceffd72e72d736 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 16:42:12 +0900
Subject: [PATCH] cifsd: spaces preferred around that '/'
Git-commit: c8ed11522b4acbe378687b6388ceffd72e72d736
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Fix "CHECK: paces preferred around that '/' from checkpatch.pl --strict.

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/asn1.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifsd/asn1.c b/fs/cifsd/asn1.c
index 479a9c1fcbbe..846f4e73abbf 100644
--- a/fs/cifsd/asn1.c
+++ b/fs/cifsd/asn1.c
@@ -65,7 +65,7 @@ static bool asn1_oid_decode(const unsigned char *value, size_t vlen,
 	unsigned long subid;
 
 	vlen += 1;
-	if (vlen < 2 || vlen > UINT_MAX/sizeof(unsigned long))
+	if (vlen < 2 || vlen > UINT_MAX / sizeof(unsigned long))
 		return false;
 
 	*oid = kmalloc(vlen * sizeof(unsigned long), GFP_KERNEL);
-- 
2.33.0

